### PR TITLE
Added License section on README.md

### LINKED ISSUE #7: Add LICENSE section
Each repo should contain a MPL LICENSE file.

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #7 

@gvn @Pomax 👀 ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ We enforce our coding standards with automated tooling ("linters"), that are usu
 
 Refer to [Mofo Style](https://github.com/MozillaFoundation/mofo-style) for more info.
 
+## License
+
+Make sure every repo contains a [MPL 2.0](https://www.mozilla.org/en-US/MPL/) `LICENSE` file. This `LICENSE` file can be added by selecting "Mozilla Public License 2.0" when you create a new repo on GitHub.
```
Comment (Pomax) [inline README.md]: small linguistic nit: "an MPL 2.0 LICENSE file" (because MPL is an acronym starting with a vowel sound)

Comment (Pomax) [inline README.md]: do we also want to say what that license option is in the `package.json`, since our projects use npm?

**Comment by gvn** [comment]
Yeah, we should probably ask people to both add the LICENSE file and also indicate the license in `package.json`'s `license` field.

**Comment by mmmavis** [comment]
PR updated! 🌞

**Comment by gvn** [comment]
Looks good, but can we change to:

For every npm-managed project, `"license" : "MPL-2.0"` should be included in the `package.json` file.

**Comment by gvn** [comment]
Good to merge w. tweak

**Comment by mmmavis** [comment]
updated + squashed.

gonna merge it now! 🍊

**Pull request merged**